hedera-node/src/main/java/com/hedera/services/store/tokens/HederaTokenStore.java

```diff
@@ -584,14 +585,26 @@ public ResponseCodeEnum update(final TokenUpdateTransactionBody changes, final l
 
 	private AliasLookup checkNewAutoRenewAccount(final TokenUpdateTransactionBody changes) {
 		if (changes.hasAutoRenewAccount()) {
-			return lookUpAccountId(changes.getAutoRenewAccount(), INVALID_AUTORENEW_ACCOUNT);
+			final var autoRenewLookUp =  lookUpAccountId(changes.getAutoRenewAccount(), INVALID_ALIAS_KEY);
+			if (autoRenewLookUp.response() != OK) {
+				return autoRenewLookUp;
+			} else {
+				final var validity = usableOrElse(autoRenewLookUp.resolvedId(), INVALID_AUTORENEW_ACCOUNT);
+				return AliasLookup.of(autoRenewLookUp.resolvedId(), validity);
+			}
 		}
 		return AliasLookup.of(AccountID.getDefaultInstance(), OK);
 	}
 
 	private AliasLookup checkNewTreasuryAccount(final TokenUpdateTransactionBody changes) {
 		if (changes.hasTreasury()) {
-			return lookUpAccountId(changes.getTreasury(), INVALID_TREASURY_ACCOUNT_FOR_TOKEN);
+			final var treasuryLookUp = lookUpAccountId(changes.getTreasury(), INVALID_ALIAS_KEY);
+			if (treasuryLookUp.response() != OK) {
+				return treasuryLookUp;
+			} else {
+				final var validity = usableOrElse(treasuryLookUp.resolvedId(), INVALID_TREASURY_ACCOUNT_FOR_TOKEN);
```

Comment:
can we do `usableOrElse` inside `lookUpAccountId`?

Reply:
that is what we did previously.